ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -252,6 +254,18 @@ Item {
             }
         }
 
+
+        Connections {
+            enabled: !!d.activeChatContentModule
+            target: d.activeChatContentModule.inputAreaModule
+            function onMessageSuccessfullySent() {
+                d.sending = false
+            }
+            function onMessageFailedToBeSent() {
```

Comment:
Do we inform the user in some way that the message failed to be sent? if not, it might be nice to handle it here.

IMO it would not be nice to spend some time to compose a nice message, try to send it. Sending fails and the message input is lost.

Reply:
I think we didn't use to, but I improved the Nim code and when I forced an error, it did show a dialog saying there was an error.
It was ugly as hell, but it worked.

It does lose the input though.
I can try to make it save the input until there is a success!

Reply:
I added a commit that makes it so we print the error in the dialog.

I wasn't able to make the input not empty itself. At least not without any ugly code.

I'll open a new issue so that it can be taken care of more carefully.

Reply:
I opened [the issue here](https://github.com/status-im/status-desktop/issues/15265) and put it for the UI team, since you guys probably have a better idea on how to keep the state of the input until the right moment, while also showing the sending indicator (we might want to have a designer take a look at it too).
I didn't put a priority or milestone. It's not top priority IMO